flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/source/reader/fetch/MongoDBStreamFetchTask.java

```diff
@@ -373,4 +377,8 @@ private OperationType getOperationType(BsonDocument changeStreamDocument) {
     private boolean isBoundedRead() {
         return !NO_STOPPING_OFFSET.equals(streamSplit.getEndingOffset());
     }
+
+    private boolean shouldEmit(ChangeStreamOffset currentOffset) {
+        return !isBoundedRead() || currentOffset.isAtOrBefore(streamSplit.getEndingOffset());
+    }
```

Comment:
This code don't be called at other place. Maybe it shoud be deleted

Reply:
Thanks a lot, I first use it to determine whether to emit, then think a unbounded stream read currentOffset is a waste, But I forget to remove it.